docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,105 @@
+=====================
+RPM rsync Distributor
+=====================
+
+Purpose:
+========
+The RPM rsync distributor publishes RPM content to a remote server. As the name suggests, the
+distributor uses rsync to copy.
+
+Configuration
+=============
+Here's example of rpm_rsync_distributor configuration:
+
+.. code-block:: json
+
+    {
+     "distributor_id": "rpm_rsync_distributor",
+     "distributor_type_id": "rpm_rsync_distributor",
+     "distributor_config": {
+        "remote": {
+            "auth_type": "publickey",
+            "ssh_login": "foo",
+            "key_path": "/home/user/.ssh/id_rsa",
+            "host": "192.168.121.1",
+            "root": "/home/foo/pulp_root_dir"
+        },
+        "predistributor_id": "yum_distributor",
+        "force_full": true
+        }
+    }
+
+``auth_type``
+  Two authentication methods are supported: ``publickey`` and ``password``.
+
+``ssh_login``
```

Comment:
As a minor suggestion, I think ``ssh_user`` would be slightly more clear, and matches the terminology used by ssh and rsync. Since "login" is often used as a verb, and since rsync can use other tools besides ssh to do remote transfers, the name ``ssh_login`` could be used to mean something besides username. I could imagine it being a boolean for whether or not to use ssh at all. But I think ``ssh_user`` is completely clear and unambiguous.